bindings/ios/MEGARequest.mm

```diff
@@ -193,4 +208,14 @@ - (NSInteger)numDetails {
     return self.megaRequest ? self.megaRequest->getNumDetails() : 0;
 }
 
+- (NSArray<MEGAStringList *> *)megaStringTableArray {
+    MegaStringTable *table = self.megaRequest ? self.megaRequest->getMegaStringTable() : nil;
```

Comment:
@jcmartinac, we don't need to check NULL here, as both `self.megaRequest` and `self.megaRequest->getMegaStringTable()` should not be NULL when the related request comes back successfully. So, `MegaStringTable *table = self.megaRequest->getMegaStringTable()` is enough here.